README.md

```diff
@@ -1,151 +1,145 @@
 
-Files (bam, vcf, fasta, bed, python)
-- Existence of file at given path
-- If BAM, existence of index file in same directory as BAM file
-- File extension type
-- Validity of file for specific file type (i.e. a vcf file is a vcf file)
-- Checksums (generates checksum comparison if .md5 or .sha512 file exists)
 
-Directories (read or read-write)
-- Existence of directory at given path
-- Readability, writability
-
-> B. Generates checksum files and compares checksum files.
-
-### input types:
-The validation action can be specified using the `-t` tag. If not specified, it defaults to `file-input`.
-
-|file types|directory types|
-|----------|---------------|
-|file-bam| directory-r |
-|file-vcf| directory-rw |
-|file-fasta|
-|file-bed|
-|file-py|
-|file-input|
-
-If an input type is specified as "file-input", it will automatically try to match the file type or simply check for existence/readability.
-If an input type is one of the checksum types, it will create a new checksum file based on the input file path.
-All input types regardless will be checked for existence.
-
-### requirements:
+**Requirements:**<br>
 When used as a standalone command line tool, the following dependencies must be installed:
 
 |tool|
 |----|
 |python 3.10|
 |vcftools 0.1.16|
 
-Otherwise, it's recommended to use the docker to keep dependencies bundled.
+## Execution Options
 
-## usage
+**Running the standalone command line tool:**
+```
+validate path/to/file.bam
+```
 
-### parameters:
+**Running as interactive docker session:**
+```
+docker run -it pipeval:3.0.0 /bin/bash
+(bash): validate path/to/file.bam
+```
+_Note: Update the tag to the latest version as necessary._
 
-Required arg
-- _path_ path of one or more files or directories to validate or generate checksum for
+**Running as Nextflow process with docker:**<br>
+See the example under [/example/](https://github.com/uclahs-cds/public-tool-PipeVal/tree/main/example) or the pipeline-align-DNA repository
 
-Optional args
-- _-t, --type_ specific input type
-- _-h, --help_ show this help message and exit
+## Validation
+The tool will attempt to automatically detect the file type and do file specific validation. 
+If the file type is unsupported, it will just do a simple existence and checksum (if MD5 or SHA512 checksums exist) check.
 
-### how to use:
+_Note: All input types will be checked for existence and checksum matching._
 
-_Running the standalone command line tool_
-```
-validate -t file-bam path/to/file.bam
-```
+### Supported Inputs
 
-_Running as interactive docker session_
-```
-docker run -it validate:1.0.0 /bin/bash
-(bash): validate -t file-input path/to/file.bam
-```
+| File Type | Description |
+| :-------: | ------ |
+| bam | Validate bam/cram/sam using `pysam`. <br> Check for an index file in same directory as the BAM.<br><br>_Note: If a BAM input is missing an accompanying BAM index file in the same directory,<br> `validate` will not throw an exception but will print a warning._|
```

Comment:
I think we want to capitalize file format names (`BAM`, `SAM`, etc) in general except file extensions (`.bam`, `.sam`)